scripts/clients.mk

```diff
@@ -0,0 +1,24 @@
+##@ Clients
```

Comment:
The main `Makefile` should only contain targets related to `build`, `install` and `test` to keep it simple and easier to maintain.

Then where do we store other targets that may be useful? I suggest that we make use of sub `Makefile`s such as `scripts/clients.mk` and `scripts/lint.mk`.

We can import the targets from these files into the main `Makefile` using `include scripts/clients.mk`. This is useful if you still want to show these targets in the main help.

It's also possible to call a target from a sub `Makefile` without going through the main `Makefile`, for example `$ make -f scripts/clients.mk geth`.

@praetoriansentry wdyt?

Reply:
I like it. I would just confirm that your changes don't impact any of the github work flows / ci.